setup.py

```diff
@@ -21,7 +21,11 @@
             "pytest==2.9.1",
         ]
     },
-    dependency_links=[],
+    entry_points={
```

Comment:
these fixtures have pretty generic names. i wonder if it's possible to make the plugin opt-in